lib/V8/v8-utils.h

```diff
@@ -26,14 +26,22 @@
 #include <stddef.h>
 #include <cstdint>
 #include <string>
-
+#include <tuple>
 #include <v8.h>
 
 #include "Basics/ErrorCode.h"
 
 namespace arangodb {
 class Result;
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief downloads data from a URL
```

Comment:
the description `downloads data from a URL` sounds strange when looking at the function name `getEndpoint`.
So either the function should be renamed or the description should be adjusted.
Also, as it is a public API now, the meaning of the parameters (and the 3 strings in the return value) should be somehow clarified.